rancid-tomatillos/src/containers/MoviePage/MoviePage.js

```diff
@@ -1,21 +1,63 @@
 import React, { Component } from 'react';
 import './MoviePage.scss';
+import StarRatingComponent from 'react-star-rating-component';
 import { connect } from 'react-redux';
 import { render } from 'react-dom';
 import { fetchRatings } from '../../apiCalls';
 import { updateUserRatings } from '../../actions';
-
+import images from '../../images/images';
 
 class MoviePage extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      rating: null
+      rating: 1
     }
```

Comment:
I am guessing this is to make it easier for someone to determine what data type the rating would be? But I think this may be a good place for a comment than a hard coded in value that is supposed to be meaningless but does have a value

Reply:
This is the way that they had it set up in the example for the star picker plugin. We should try changing it back to null now that it is working and see if it still works. If so, then keeping it at null would be ideal, but it might be the case that it needs to start at 1 for that functionality to work correctly.